app/models/manageiq/providers/storage_manager/cinder_storage_manager/event_parser.rb

```diff
@@ -0,0 +1,29 @@
+#
+# TODO: (hsong) duplicate with ManageIQ::Providers::NetworkManager::EventParser
+#                   and ManageIQ::Providers::InfraManager::EventParser
+#
+module ManageIQ::Providers::StorageManager::CinderStorageManager::EventParser
```

Comment:
I don't know whether Cinder generate events or not, but seems like all kind of providers have their own event parser. If you sure we don't need it, I can remove it.